### PR TITLE
Cache metadata from fedora-packages

### PR DESCRIPTION
**Pull request by ralphbean**
... in our own db to make us faster.

### CODE REVIEW

**Comment by ralphbean** [comment]
Should fix #190.

File: createdb.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
 import sys
 
 if '--with-dev-data' in sys.argv:
-    package = model.Package(id=1, name=u"mattd", summary=u"Matt Daemon ☃")
```
Comment (lmacken) [inline createdb.py]: RIP mattd.

**Comment by lmacken** [comment]
Looks good to me, :+1:

**Comment by pypingou** [comment]
Do we want a way to invalidate this cache? Otherwise :+1:

File: createdb.py (hunk 1 above)
Comment (ralphbean) [inline createdb.py]: :ghost: :dead:

**Comment by ralphbean** [comment]
> Do we want a way to invalidate this cache? Otherwise :+1:

We'll need that eventually, but right now it doesn't change in fedora-packages (it may end up changing after the refactoring in ~Q3).

Thanks guys!

**Pull request merged**